### ISSUE TITLE
when i compile a mac app from windows: "There is a problem with symlink"

### ISSUE THREAD

**Issue by WI-FA** [nwjs-community/nw-builder#463]
I'm trying to compile an app for mac since windows, and there is an error about "symlink", besides when i try to do it since mac, the app crashes  in other macs, so what  can i do, about it?

¡Thanks!

**Comment by carrotalan** [comment]
I also get an error when I have a dependancy on the project which is stored locally on the computer and I choose the zip:false option:

package.json
```
  "dependencies": {
    "kiosk-client-api": "file:../kiosk-client-api"
  },
```

Error during build
`Error: EINVAL: invalid argument, readlink '/Volumes/Data/Users/work/git/kiosk-platform/kiosk-client/build/app/kiosk-client/win32/node_modules/kiosk-client-api'`

This does not happen if I delete the symlink and copy the real folder inside

**Comment by adam-lynch** [comment]
@WI-FA Does it work without nw-builder? See https://github.com/nwjs/nw.js/wiki/How-to-run-apps on how to try that. It involves getting an official NW.js executable and dropping your project directory on the NW.js icon.

**Comment by Blatman** [comment]
This has probably been solved but I had a symlink issue when building an OSX app on Win10 VM.
Failure during download of NWjs OSX version -
Uncaught (in promise) Error: EPERM: operation not permitted, symlink 'Versions\Current\Helpers'  -> ....

According to other sources on the net node needs Administrative rights to handle symlinks.
 I added the User account to the Local Policy (Create symbolic links), restarted the VM and bingo the symlink handling worked OK.

For more info on tweaking the policies -
https://superuser.com/questions/104845/permission-to-make-symbolic-links-in-windows-7

Note: this may also help if using NWB with the CLI mode - I just happen to be using a GUI to run NWB.

**Comment by IceReaper** [comment]
I had the same problem on a different project. The resolution was to use "junction" instead of "dir" on the symlinks as third parameter when creating them. Then windows did not require admin rights anymore.

**Comment by djmisterjon** [comment]
> 

yes , config your window env correctly is needed , by default window is fully restrictive.
force elevation
https://www.ghacks.net/2016/12/04/windows-10-creators-update-symlinks-without-elevation/
and be sure you also activate the developper mode.
![image](https://user-images.githubusercontent.com/24865815/169157190-23d54a8d-a35d-4ffd-ade0-b734f2c703a6.png)
